ignite/services/plugin/plugin_test.go

```diff
@@ -353,6 +362,104 @@ func TestPluginLoad(t *testing.T) {
 	}
 }
 
+func TestPluginLoadSharedHost(t *testing.T) {
+	// wd, err := os.Getwd()
+	// require.NoError(t, err)
+
+	// scaffoldPlugin runs Scaffold and updates the go.mod so it uses the
+	// current ignite/cli sources.
+	scaffoldPlugin := func(t *testing.T, dir, name string) string {
+		require := require.New(t)
+		path, err := Scaffold(dir, name, true)
+		require.NoError(err)
+		// We want the scaffolded plugin to use the current version of ignite/cli,
+		// for that we need to update the plugin go.mod and add a replace to target
+		// current ignite/cli
+		gomod, err := gomodule.ParseAt(path)
+		require.NoError(err)
+		// use GOMOD env to get current directory module path
+		modpath, err := gocmd.Env(gocmd.EnvGOMOD)
+		require.NoError(err)
+		modpath = filepath.Dir(modpath)
+		gomod.AddReplace("github.com/ignite/cli", "", modpath, "")
+		// Save go.mod
+		data, err := gomod.Format()
+		require.NoError(err)
+		err = os.WriteFile(filepath.Join(path, "go.mod"), data, 0o644)
+		require.NoError(err)
+		return path
+	}
```

Comment:
Let's not defined this function twice (here and in `TestPluginLoad`), let's define it package wide.